src/src/Helpers/Modbus_device.cpp

```diff
@@ -48,6 +47,16 @@ bool ModbusDEVICE_struct::init(uint8_t                 slaveAddress,
 
   _modbus_address = slaveAddress;
 
+  if (loglevelActiveFor(LOG_LEVEL_INFO)) {
+    String log = F("---> ModbusDevice Init: Slave address = ");
+    log += slaveAddress;
+    log += F(", This = ");
+    log += (ulong)this;
```

Comment:
Why does this compile? And what is an `ulong`?
In C++ there is a special integer type which is by definition large enough to hold a pointer. This is the `size_t`
On our ESP boards, this happens to be 32-bit in size, so essentially it is the same as an `uint32_t`, which happens to be the same as `unsigned int`.
Though it is good practice to use `size_t` for these kind of things.